### ISSUE TITLE
Question: Adding custom NPCs

### ISSUE THREAD

**Issue by madhandennis** [azerothcore/mod-guildhouse#52]
Hi,

Is it possible to add custom NPCs without modifying the code? For example, I would like to add the Assistant NPC from https://github.com/noisiver/mod-assistant. Doing a `.npc add 9000000` doesn't seem to work. Any suggestions?

Thanks!

**Comment by mlsgames-mschenck** [comment]
yes but you have to modify the sql in acore_world.  you can do .npc add temp 9000000 but every time the server restarts you will have to add her back.

Here is the sql to add the Assistant NPC, i put her on the top floor near the secondary skill trainers.  i am also attaching a file with a number of other NPC that i added as well as a cookpot as i could not find one.
[GuildHouseCustomObjects.txt](https://github.com/user-attachments/files/16587307/GuildHouseCustomObjects.txt)

insert into acore_world.creature (id1,id2,id3,map,zoneId,areaId,spawnMask,phaseMask,equipment_id,position_x,position_y,position_z,orientation,spawntimesecs,wander_distance,currentwaypoint,curhealth,curmana,MovementType,npcflag,unit_flags,dynamicflags,ScriptName,VerifiedBuild,CreateObject,Comment) values (9000000,0,0,1,0,0,1,11,0,16224.19,16303.937,29.262074,6.22119,300,0,0,10635,0,0,0,0,0,'',null,0,'Assistant');
